docs/notes/2.23.x.md

```diff
@@ -60,6 +60,8 @@ The backend linter will also load a Trufflehog [configuration file](https://gith
 
 #### Python
 
+[The `pants.backend.python.lint.black` backend](https://www.pantsbuild.org/2.23/reference/subsystems/black) now uses version 24.4.2 by default, upgrading from 23.3.0. This comes with two notable visible changes: there's a new stable style (see [release notes for 24.1.0](https://black.readthedocs.io/en/stable/change_log.html) for details) which may result in extensive reformatting, and black now requires Python 3.8 or newer to _run_ (but can still format code from older versions). To override Pants' default version, use [the `install_from_resolve` option](https://www.pantsbuild.org/2.23/reference/subsystems/black#install_from_resolve) and/or [the `interpreter_constraints` option](https://www.pantsbuild.org/2.23/reference/subsystems/black#interpreter_constraints).
```

Comment:
> `(see [release notes for 24.1.0](https://black.readthedocs.io/en/stable/change_log.html) for details)`

NB. this doesn't link to the release notes heading directly because they're just numbered links from the top, not permalinks, e.g. https://black.readthedocs.io/en/stable/change_log.html#id18 

This means a direct link will be wrong/misleading by the time we release Pants 2.23, if black does any releases in that time, since the 18th heading won't be 24.1.0 any more.

https://github.com/psf/black/issues/4344